### PR TITLE
[RHOAIENG-591] - Add InstascaleMachineSet test 

### PR DESCRIPTION
**Pull request by ChughShilpa**
[RHOAIENG-591](https://issues.redhat.com/browse/RHOAIENG-591)

### CODE REVIEW

File: ods_ci/tests/Resources/Files/machineset-template.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (sutaakar) [inline ods_ci/tests/Resources/Files/machineset-template.sh:1]: Have you considered creating Machine set from scratch as described in https://docs.openshift.com/container-platform/4.11/machine_management/creating_machinesets/creating-machineset-aws.html ?

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot [hunk 2]
```diff
@@ -120,3 +152,11 @@ Run Codeflare ODH Test
     IF    ${result.rc} != 0
         FAIL    ${TEST_NAME} failed
     END
+
+Enable Instascale
+    Log To Console    "Enable instascale to true in config map ....."
+    ${result} =    Run Process    oc get cm codeflare-operator-config -n redhat-ods-applications -o yaml | sed -e 's|enabled: false|enabled: true|' | oc apply -f -
```
Comment (sutaakar) [inline ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot:158]: AFAIK the CodeFlare operator needs to be restarted after ConfigMap change, to spin new pod with updated ConfigMap.

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot [hunk 3]
```diff
@@ -56,6 +57,36 @@ Run TestMnistPyTorchMCAD ODH test
...
+    Log To Console    "Create machine set ....."
+    ${result} =    Run Process    sh    ods_ci/tests/Resources/Files/machineset-template.sh
+    ...    shell=true    stderr=STDOUT
+    IF    ${result.rc} != 0
+        FAIL    Can not create machine set
+    END
+
+    # Delete the existing GPU machineset
+    # This needs to be done as part of test execution and a corresponding jira is tracked here - https://issues.redhat.com/browse/RHOAIENG-1118
+    ${result} =    Run Process    oc get machineset -n openshift-machine-api -o name | head -n1
```
Comment (sutaakar) [inline ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot:82]: This looks suspicious. What Machine set do you delete? What is the deleted Machine set used for?

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot [hunk 4]
```diff
@@ -56,6 +57,36 @@ Run TestMnistPyTorchMCAD ODH test
...
+    [Tags]  Instascale
+    ...     Tier3
+    ...     DistributedWorkloads
+    ...     CodeflareOperator
+    Skip If RHODS Is Managed
+
+    # Enable instascale to true in the codeflare operator config map
+    Enable Instascale
+
+    # Create machine set
```
Comment (sutaakar) [inline ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot:72]: Can you remove the Machine set in teardown, to cleanup the environment?

**Comment by ChughShilpa** [comment]
Closed as the required jira is closed now - https://issues.redhat.com/browse/RHOAIENG-591

**Pull request closed without merging**